rulesets/rulesets_test.go

```diff
@@ -353,6 +355,17 @@ rules:
 
 }
 
+func TestRuleSetsModel_GenerateRuleSetFromConfig_Oas_Owasp(t *testing.T) {
+
+	yaml := `extends: [[spectral:oas, all], [spectral:owasp, all]]`
```

Comment:
I know this is a little beyond this PR, but the more I see 'spectral' in these rulesets, the more I want to establish `vacuum` as a namespace.

Would you be able to add `vacuum:owasp` as an alias to `spectral:owasp`?

Reply:
Done